Tasks/Voleyko/Task2.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+from datetime import time
+
+d_h = {00:('двенадцать часов'),1:('Один Час'),2:('Два часа'),3:('Три часа'),4:('Четыре часа'),
+5:('Пять часов'), 6:('Шесть часов'),7:('Семь часов'),8:('Восемь часов'),
+9:('Девять часов'), 10:('Десять часов'),11:('Одиннадцать часов'),12:('Двенадцать часов')}
+
+d_m = {00:'ровно',1:'одна минута', 2:'две минуты', 6:'шесть минут', 7:'семь минут', 8:'восемь минут',9:'девять минут',10:'десять минут',11:'одиннадцать минут',12:'двенадцать минут',
+13:'тренадцать минут',14:'четырнадцать минут',15:'пятнадцать минут',16:'шестьнадцать минут',17:'семьнадцать минут',18:'восемьнадцать минут',19:'девятнадцать минут',20:'двадцать минут',21:'двадцать одна минута',22:'двадцать две минуты',23:'двадцать три минуты',24:'двадцать четыре минуты',
+25:'двадцать пять минут',26:'двадцать шесть минут',27:'двадцать семь минут',28:'двадцать восемь минут',29:'двадцать девять минут',30:'тридцать минут',31:'дридцать одна минута',32:'тридцать две минуты',33:'тридцать три минуты',34:'тридцать четыре минут',35:'тридцать пять минут',36:'тридцать шесть минут',37:'тридцать семь минут',38:'тридцать восемь минут',
+39:'тридцать девять минут',40:'сорок минут',41:'сорок одна минута',42:'сорок две минуты',43:'сорок три минуты',44:'сорок четыре минуты',45:'сорок пять минут',46:'сорок шесть минут',47:'сорок семь минут',48:'сорок восемь минут',49:'сорок девять минут',50:'пятьдесят минут',51:'петьдесят одна минута',52:'пятьдесят две минуты',53:'пятьдесят три минуты',54:'пятьдесят четыре минуты',55:'пятьдесят пять минут',
+56:'пятьдесят шесть минут', 57:'пятьдесят семь минут',58:'пятьдесят восемь минут',59:'пятьдесят девять минут'}
```

Comment:
I've already seen such variable names somewhere =) 

Please make sure to use more meaningful names in the future, the data structure is good enough, but it's possible to make it better by eliminating the minutes tokens